backend/pick-git/src/main/java/com/woowacourse/pickgit/user/presentation/dto/request/ProfileDescriptionRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.woowacourse.pickgit.user.presentation.dto.request;
+
+import javax.validation.constraints.NotNull;
+
+public class ProfileDescriptionRequest {
+
+    @NotNull
+    private String description;
```

Comment:
```suggestion
    @NotNull
    @Size(max = 160)
    private String description;
```

한 줄 소개 길이 제한(+ 테스트) 추가해주세요!